Source/Blazorise/Fluent/FluentColumn.cs

```diff
@@ -211,14 +214,41 @@ private class ColumnDefinition
     #region Methods
 
     /// <inheritdoc/>
-    public string Class( IClassProvider classProvider )
+    public string Class( IClassProvider classProvider, IRowContext rowContext, IColumnableComponent currentColumnable )
     {
         if ( dirty )
         {
             void BuildClasses( ClassBuilder builder )
```

Comment:
Why use Closures here.
Maybe `ClassBuilder` should be able to be used without an `Action` when applicable? Meaning that you would remove this local function and just run the logic regularly using the new `ClassBuilder` instance you have built. Similar to how you use a regular `StringBuilder`.

Something similar to this?
![image](https://user-images.githubusercontent.com/22283161/206876084-94126731-58b1-4e76-a510-9568f824f52b.png)

Reply:
To add to this, if `Dirty` is called alot, might be beneficial to just have a `ClassBuilder` member in the `FluentColumn`

Reply:
I guess we had some other refactoring issue open for something similar to this, that we haven't gotten around to yet I think. I just happened to stumble upon this. Let me see if I find it and copy paste this there.

Reply:
Oh you just opened an issue. Kk